lib/src/themes/paint_factory.dart

```diff
@@ -12,13 +12,15 @@ class PaintStyle {
   final Expression<double> opacity;
   final Expression<double> strokeWidth;
   final Expression<Color> color;
+  final List<num> dashArray;
```

Comment:
let's name this what it is - e.g. `strokeDashPattern`

Reply:
done.